tests/visual/percy-storybook/DashTable.percy.tsx

```diff
@@ -359,4 +359,117 @@ storiesOf('DashTable/Without id', module)
+        setProps={setProps}
+        id='table'
+        data={sparseData}
+        columns={mergedColumns}
+        sort_action={TableAction.Native}
+        sort_by={[{ column_id: 'a', direction: 'asc' }]}
+        style_data_conditional={style_data_conditional}
+    />))
+    .add('"a" descending', () => (<DataTable
+        setProps={setProps}
+        id='table'
+        data={sparseData}
+        columns={mergedColumns}
+        sort_action={TableAction.Native}
+        sort_by={[{ column_id: 'a', direction: 'desc' }]}
+        style_data_conditional={style_data_conditional}
+    />))
+    .add('"a" ascending -- empty string override', () => (<DataTable
+        setProps={setProps}
+        id='table'
+        data={sparseData}
+        columns={mergedColumns}
+        sort_action={TableAction.Native}
+        sort_by={[{ column_id: 'a', direction: 'asc' }]}
+        sort_as_null={['']}
+        style_data_conditional={style_data_conditional}
+    />))
+    .add('"a" descending -- empty string override', () => (<DataTable
+        setProps={setProps}
+        id='table'
+        data={sparseData}
+        columns={mergedColumns}
+        sort_action={TableAction.Native}
+        sort_by={[{ column_id: 'a', direction: 'desc' }]}
+        sort_as_null={['']}
+        style_data_conditional={style_data_conditional}
+    />))
+    .add(`"a" descending -- '' & 426 override`, () => (<DataTable
+        setProps={setProps}
+        id='table'
+        data={sparseData}
+        columns={mergedColumns}
+        sort_action={TableAction.Native}
+        sort_by={[{ column_id: 'a', direction: 'desc' }]}
+        sort_as_null={['', 426]}
+        style_data_conditional={style_data_conditional}
+    />))
+    .add(`"a" ascending -- '' and 426 override`, () => (<DataTable
+        setProps={setProps}
+        id='table'
+        data={sparseData}
+        columns={mergedColumns}
+        sort_action={TableAction.Native}
+        sort_by={[{ column_id: 'a', direction: 'asc' }]}
+        sort_as_null={['', 426]}
+        style_data_conditional={style_data_conditional}
+    />));
+
+storiesOf('DashTable/Export', module)
+        .add('Export Button for xlsx file', () => (<DataTable
+            setProps={setProps}
+            data={dataA2J}
+            columns={columnsA2J}
+            export_format= {'xlsx'}
+        />))
+        .add('Export Button for csv file', () => (<DataTable
+            setProps={setProps}
+            data={dataA2J}
+            columns={columnsA2J}
+            export_format= {'xlsx'}
+        />))
+        .add('No export Button for file formatted not supported', () => (<DataTable
+            setProps={setProps}
+            data={dataA2J}
+            columns={columnsA2J}
+            export_format= {'json'}
+        />))
+        .add('No export Button', () => (<DataTable
+            setProps={setProps}
+            data={dataA2J}
+            columns={columnsA2J}
+            export_format= {'none'}
+        />));
```

Comment:
I think we can move those new tests to their own file. The 100 rows of data is not super useful here - you can cut it down to like ~10 or something for these.

Reply:
removing obsolete comment!